ansible/roles/nginx/tasks/main.yml

```diff
@@ -194,6 +203,7 @@
 
   - name: Update env name in aaad-js file
     lineinfile: dest="{{nginx_conf_dir}}/aaad/static/aaad-ui.js" regexp="^\$env_name = " line="$env_name = '{{ env_name }}'"
+    when: ansible_check_mode is not defined or not ansible_check_mode
```

Comment:
Yes it was and the very first time nginx was deployed on a node, this would break the play.